### ISSUE TITLE
A11y: northstar_application_navigation_buttons  Modal contrast

### ISSUE THREAD

**Issue by shane-exygy** [civiform/civiform#10413]
**Describe the bug**

  1) src/applicant/navigation/northstar_application_navigation_buttons.test.ts:209:11 › Applicant navigation flow › navigation with five blocks › previous button › clicking previous with some missing answers shows error modal @northstar › Validate accessiblity 

    Error: Found 1 axe accessibility violations
    On page: http://civiform:9000/programs/2/blocks/1/false/PREVIOUS_BLOCK

    expect(received).toEqual(expected) // deep equality

    - Expected  -  1
    + Received  + 91

    - Array []
    + Array [
    +   Object {
    +     "description": "Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds",
    +     "help": "Elements must meet minimum color contrast ratio thresholds",
    +     "helpUrl": "https://dequeuniversity.com/rules/axe/4.10/color-contrast?application=playwright",
    +     "id": "color-contrast",
    +     "impact": "serious",
    +     "nodes": Array [
    +       Object {
    +         "all": Array [],
    +         "any": Array [
    +           Object {
    +             "data": Object {
    +               "bgColor": "#ffffff",
    +               "contrastRatio": 4.09,
    +               "expectedContrastRatio": "4.5:1",
    +               "fgColor": "#3b83b8",
    +               "fontSize": "12.7pt (16.96px)",
    +               "fontWeight": "bold",
    +               "messageKey": null,
    +             },
    +             "id": "color-contrast",
    +             "impact": "serious",
    +             "message": "Element has insufficient color contrast of 4.09 (foreground color: #3b83b8, background color: #ffffff, font size: 12.7pt (16.96px), font weight: bold). Expected contrast ratio of 4.5:1",
    +             "relatedNodes": Array [
    +               Object {
    +                 "html": "<div class=\"usa-modal cf-modal\" tabindex=\"-1\">",
    +                 "target": Array [
    +                   ".cf-modal",
    +                 ],
    +               },
    +             ],
    +           },
    +         ],
    +         "failureSummary": "Fix any of the following:
    +   Element has insufficient color contrast of 4.09 (foreground color: #3b83b8, background color: #ffffff, font size: 12.7pt (16.96px), font weight: bold). Expected contrast ratio of 4.5:1",
    +         "html": "<button type=\"button\" data-redirect-to=\"/programs/2/review\" data-close-modal=\"\" class=\"usa-button usa-button--outline margin-y-2\">Go to the previous page without saving</button>",
    +         "impact": "serious",
    +         "none": Array [],
    +         "target": Array [
    +           "button[data-redirect-to=\"/programs/2/review\"]",
    +         ],
    +       },
    +       Object {
    +         "all": Array [],
    +         "any": Array [
    +           Object {
    +             "data": Object {
    +               "bgColor": "#3b83b8",
    +               "contrastRatio": 4.09,
    +               "expectedContrastRatio": "4.5:1",
    +               "fgColor": "#ffffff",
    +               "fontSize": "12.7pt (16.96px)",
    +               "fontWeight": "bold",
    +               "messageKey": null,
    +             },
    +             "id": "color-contrast",
    +             "impact": "serious",
    +             "message": "Element has insufficient color contrast of 4.09 (foreground color: #ffffff, background color: #3b83b8, font size: 12.7pt (16.96px), font weight: bold). Expected contrast ratio of 4.5:1",
    +             "relatedNodes": Array [
    +               Object {
    +                 "html": "<button type=\"button\" data-close-modal=\"\" class=\"usa-button margin-y-2\">Stay here and fix your answers</button>",
    +                 "target": Array [
    +                   ".margin-y-2[data-close-modal=\"\"][type=\"button\"]:nth-child(4)",
    +                 ],
    +               },
    +             ],
    +           },
    +         ],
    +         "failureSummary": "Fix any of the following:
    +   Element has insufficient color contrast of 4.09 (foreground color: #ffffff, background color: #3b83b8, font size: 12.7pt (16.96px), font weight: bold). Expected contrast ratio of 4.5:1",
    +         "html": "<button type=\"button\" data-close-modal=\"\" class=\"usa-button margin-y-2\">Stay here and fix your answers</button>",
    +         "impact": "serious",
    +         "none": Array [],
    +         "target": Array [
    +           ".margin-y-2[data-close-modal=\"\"][type=\"button\"]:nth-child(4)",
    +         ],
    +       },
    +     ],
    +     "tags": Array [
    +       "cat.color",
    +       "wcag2aa",
    +       "wcag143",
    +       "TTv5",
    +       "TT13.c",
    +       "EN-301-549",
    +       "EN-9.1.4.3",
    +       "ACT",
    +     ],
    +   },
    + ]

      225 |         )
      226 |
    > 227 |         await validateAccessibility(page)
          |                                    ^
      228 |         await validateScreenshot(
      229 |           page,
      230 |           'northstar-error-on-previous-modal',
        at /usr/src/civiform-browser-tests/src/applicant/navigation/northstar_application_navigation_buttons.test.ts:227:36

    attachment #2: trace (application/zip) ─────────────────────────────────────────────────────────
    tmp/test-output/applicant-navigation-north-42b5c-g-answers-shows-error-modal/trace.zip
    Usage:

        npx playwright show-trace tmp/test-output/applicant-navigation-north-42b5c-g-answers-shows-error-modal/trace.zip

    ────────────────────────────────────────────────────────────────────────────────────────────────

  1 failed
    src/applicant/navigation/northstar_application_navigation_buttons.test.ts:209:11 › Applicant navigation flow › navigation with five blocks › previous button › clicking previous with some missing answers shows error modal @northstar 

**To Reproduce**

`bin/run-browser-tests northstar_application_navigation_buttons.test.ts:209`

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Mac

**Smartphone (please complete the following information):**

**Additional context**

**Comment by shane-exygy** [comment]
This appears to have been a local flake
